src/renderer/routes/swap.ts

```diff
@@ -10,10 +10,13 @@ export const base: Route<void> = {
 
 export type SwapRouteParams = { source: string; target: string }
 export const swap: Route<SwapRouteParams> = {
-  template: `${base.template}/:source-:target`,
+  /**
+   * Use '--' 'cause asset symbols have '-' separator
+   */
+  template: `${base.template}/:source--:target`,
```

Comment:
> Use '--' 'cause asset symbols have '-' separator

[suggestion] How about to use a `|` (pipe)? 

```
BNB.TUSDB-000--BNB.RUNE-67C
```
vs.
```
BNB.TUSDB-000|BNB.RUNE-67C
```

Reply:
not a problem at all :) just a convo here